tox.ini

```diff
@@ -41,15 +41,15 @@ whitelist_externals = bash
 skip_install = True
 deps =
     flake8
-    flake8-import-order>=0.9
+    flake8-import-order==0.10
 
 [testenv:py27-lint-imports-include-list]
 commands = bash .ci/flake8_wrapper_imports.sh
 whitelist_externals = bash
 skip_install = True
 deps =
     flake8==3.0.4
-    flake8-import-order>=0.9
+    flake8-import-order==0.10
```

Comment:
I think you need to go back to `0.9` and also fix this error:
```
tools/filters/gff/gff_filter_by_attribute.py:10:1: I100 Import statements are in the wrong order.from ast should be before from json
```
Then all tests should pass.